azure/route.sh

```diff
@@ -12,6 +12,34 @@ fn azure_route_table_delete(name, group) {
 	)
 }
 
+fn azure_route_table_route_new(name, group, routetable, address, hoptype) {
+	instance = (
+		"--name"
+		$name
+		"--resource-group"
+		$group
+		"--route-table-name"
+		$routetable
+	    "--address-prefix"
+		$address
+	    "--next-hop-type"
+		$hoptype
+	)
+
+	return $instance
+}
+
+fn azure_route_table_route_set_hop_address(instance, hopaddress) {
```

Comment:
To create an internet route you need execute the command with "--next-hop-type" set to "Internet"

azure network route-table route create --resource-group <resource-group> --route-table-name <route-table-name> --name <route-name> --next-hop-type Internet

I can't to pass the "--next-hop-ip-address" options for this case.

To create a route to a specific host, an example it is the route to NAT, you use "--next-hop-type" set to "VirtualAppliance".
In this case we need set the "--next-hop-ip-address" with the host ip address (NAT).

I don't know if you can pass more than one ip address. I don't think so.